process/block/preprocess/transactions.go

```diff
@@ -829,7 +831,17 @@ func (txs *transactions) createAndProcessMiniBlocksFromMe(
 		if len(miniBlock.TxHashes) == 0 {
 			numNewMiniBlocks = 1
 		}
-		if isMaxBlockSizeReached(numNewMiniBlocks, 1) {
+		numNewTxs := 1
+
+		isCrossShardScCall := receiverShardID != txs.shardCoordinator.SelfId() && core.IsSmartContractAddress(tx.RcvAddr)
+		if isCrossShardScCall {
+			if !firstCrossShardScCallFound {
+				numNewMiniBlocks++
+			}
+			numNewTxs += 1 * core.MultiplyFactorForScCall
```

Comment:
1*core.MultiplyFactorForScCall 
why 1\*x instead of simply x ?

Is it supposed to be changed later to a different value? then use a constant.

Reply:
I just kept an old pattern when 1 was a variable named numTxs. I will change.